src/styl/imports/colors.styl

```diff
@@ -0,0 +1,26 @@
+primary-color = #153e68
+secondary-color = #3faafb
+tertiary-color = #2868cb
+tertiary-hover-color = darken(tertiary-color, 20%)
+white-color = #ffffff
+grey-color = #ededed
+grey-dark-color = #bcbcbc
+black-color = #000000
+
+link-color = #67d0a1
+link-hover-color = lightness(link-color, 80%)
+
+// TAGS
```

Comment:
Is there a way we could automatically generate colors for these tags instead of hardcoding it?

Reply:
Do you have any ideas about how to implement this ? I'm not sure

Reply:
I love it! Well done!